board/flasher.h

```diff
@@ -26,23 +26,17 @@ int usb_cb_control_msg(USB_Setup_TypeDef *setup, uint8_t *resp, bool hardwired)
       break;
     // **** 0xb1: unlock flash
     case 0xb1:
-      if (FLASH->CR & FLASH_CR_LOCK) {
-        FLASH->KEYR = 0x45670123;
-        FLASH->KEYR = 0xCDEF89AB;
+      if (flash_unlock()) {
```

Comment:
maybe let's split this in two functions instead? `flash_is_locked()` and `flash_unlock()`?